### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
List of changes:

- Use common source url
- Fix missing output
- Badges and more badges!
- Add changelog to html doc
- Always use latest ex_doc
- Add changelog link to hex module info page
- Fix typos
- Revise installation doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # Doctor
 
-[![Hex.pm](https://img.shields.io/hexpm/v/doctor.svg)](http://hex.pm/packages/doctor) [![Build
-Status](https://travis-ci.org/akoutmos/doctor.svg?branch=master)](https://travis-ci.org/akoutmos/doctor) [![Coverage
-Status](https://coveralls.io/repos/github/akoutmos/doctor/badge.svg?branch=master)](https://coveralls.io/github/akoutmos/doctor?branch=master)
+[![Build Status](https://travis-ci.org/akoutmos/doctor.svg?branch=master)](https://travis-ci.org/akoutmos/doctor)
```
Comment (akoutmos) [inline README.md:3]: Switch the list around so that the module version is the first thing.

**Review by akoutmos — CHANGES_REQUESTED** [review]
Thanks for submitting a PR with all these goodies in it! Just one small nitpick and it's good to merge :)

File: README.md (hunk 1 above)
Comment (kianmeng) [reply README.md:4]: > Switch the list around so that the module version is the first thing.

Done. :smile:

**Comment by akoutmos** [comment]
Thanks for the contribution!! Appreciate it :).

**Pull request merged**